draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss

```
@@ -61,23 +62,30 @@ $border-color: rgba($color-gray-600-rgb, 0.2);
 
 .button {
```

```suggestion
.header {
```

Nitpick, but it's not a button anymore, so this was a little confusing.